modules/import-export/mod-mp3/ExportMP3.cpp

```diff
@@ -181,17 +181,15 @@ static const TranslatableStrings varModeNames {
 };
 */
 static const TranslatableStrings setRateNames {
-   /* i18n-hint: Slightly humorous - as in use an insane precision with MP3.*/
-   XO("Insane, 320 kbps"),
-   XO("Extreme, 220-260 kbps"),
+   XO("Highest, 320 kbps"),
+   XO("High, 220-260 kbps"),
```

Comment:
I disapprove of calling these "highest" and "high". MP3 becomes transparent (=a listener cannot reliably differentiate between the source and the encoded content) at around 192 kbps, depending on content of course. So for most content, "Standard" actually is the highest quality while giving you the smallest file sizes, and everything above that is just wasteful.